.github/workflows/Params.yml

```diff
@@ -0,0 +1,69 @@
+name: Params
+
+on:
+  workflow_call:
+    inputs:
+      python_version:
+        description: 'Python version.'
+        required: false
+        default: '3.10'
+        type: string
+      python_version_list:
+        description: 'Space separated list of Python versions to run tests with.'
+        required: false
+        default: '3.6 3.7 3.8 3.9 3.10'
+        type: string
+      name:
+        description: 'Name of the tool.'
+        required: true
+        type: string
+    outputs:
+      params:
+        description: "Parameters to be used in other jobs."
+        value: ${{ jobs.Params.outputs.params }}
+      python_jobs:
+        description: "List of Python versions to be used in the matrix of other jobs."
+        value: ${{ jobs.Params.outputs.python_jobs }}
+
+jobs:
+
+  Params:
+    runs-on: ubuntu-latest
+    outputs:
+      params: ${{ steps.params.outputs.params }}
+      python_jobs: ${{ steps.params.outputs.python_jobs }}
+    steps:
+
+      - name: Generate 'params' and 'python_jobs'
+        id: params
+        shell: python
+        run: |
+          name = '${{ inputs.name }}'
+          params = {
+            'package': name,
+            'python_version': '${{ inputs.python_version }}',
+            'artifacts': {
+              'coverage': f'{name}-coverage',
+              'typing': f'{name}-typing',
+              'wheel': f'{name}-wheel',
+              'doc': f'{name}-doc',
+            }
+          }
+          print(f'::set-output name=params::{params!s}')
+          print("Params:")
+          print(params)
+
+          data = {
+             '3.6': { 'icon': '🔴', 'until': '23.12.2021' },
```

Comment:
It is currently not used. However, it takes almost the same space to have it as a comment or to have it actually included in the object/dictionary. When we deprecate 3.6 in a month, instead of removing it, we can actually print a warning message using the `until` field.

Reply:
Nice idea.